configuration_broker/doc/broker.puml

```diff
@@ -0,0 +1,58 @@
+@startuml
```

Comment:
What is this file?

Reply:
Its the source file for the diagram in the readMe - using https://plantuml.com/.

We use it a lot in our docs as in means you can easily keep the source of diagrams in the repo and the syntax is pretty easy to read and update.   We haven't found anything else that covers the same scope.

I can add a comment in there explaining ?

Reply:
We use Mermaid for the same thing, GitHub renders it as inline diagram blocks in Markdown.

Reply:
I can try converting it to Mermaid if you like, I just found it a bit limited compared to plantUML in the past,

Reply:
I can try and covert to Mermaid if you want - I've always found it rather limited compared to plantuml